InfiniTAM/CMakeLists.txt

```diff
@@ -85,12 +85,12 @@ message(INFO "OpenNI found: " ${OPENNI_FOUND})
 
 add_subdirectory(Engine)
 
-cd_add_executable(InfiniTAM_cli InfiniTAM_cli.cpp)
+add_executable(InfiniTAM_cli InfiniTAM_cli.cpp)
```

Comment:
it would have been `cs_add_executable` not `cd_add_executable`

Try to address requested changes and not just merge.